### PR TITLE
Added Discord OAuth provider

### PR DESCRIPTION
**Pull request by armino-dev**
In reference to #5 
I have finished the implementation of Discord OAuth provider.
I hope this is ok.

### CODE REVIEW

File: app/config/providers.php [hunk 1]
```diff
@@ -85,7 +85,11 @@
         'enabled' => true,
         'mock' => false,
     ],
-
+    'discord' => [
+        'developers' => 'https://discordapp.com/developers/docs/topics/oauth2',
+        'icon' => 'icon-discord',
+        'enabled' => true,
```
Comment (eldadfux) [inline app/config/providers.php]: Just need to add here the mock>false attribute. It's new just added it today.

Comment (armino-dev) [reply app/config/providers.php]: ok, I'll do that

**Comment by eldadfux** [comment]
Seems like a great work!

Wrote just one small comment about a change I did today.

Also, please confirm you have tested this manually and you have been able to successfully login using this provider.

**Comment by eldadfux** [comment]
THANK YOU! All changes merged 🥳 

Please reach out to me on our [Discord server](https://discord.gg/GSeTUeA) if you would like me to send you some cool Appwrite stickers and magnets!

As a way of saying thank you, we have just invited you to join the Appwrite organization on GitHub. You can accept the invite by visiting https://github.com/orgs/appwrite/invitation. By joining our team, you will officially be an Appwrite maintainer on GitHub.

You can change your membership visibility settings, so your new Appwrite team membership badge will show up on your personal GitHub profile.

**Pull request merged**